calculator/src/test/java/com/example/calculator/scoring/ScoringRulesTest.java

```diff
@@ -0,0 +1,212 @@
+package com.example.calculator.scoring;
+
+import com.example.calculator.dto.EmploymentDto;
+import com.example.calculator.dto.ScoringDataDto;
+import com.example.calculator.enums.EmploymentStatusEnum;
+import com.example.calculator.enums.GenderEnum;
+import com.example.calculator.enums.MaritalStatusEnum;
+import com.example.calculator.enums.PositionEnum;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.env.Environment;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class ScoringRulesTest {
+
+    @Value("${baseRate:10.0}") // Значение по умолчанию
+    private BigDecimal baseRate;
```

Comment:
Private field 'baseRate' is assigned but never accessed
Field can be converted to a local variable